cfme/services/catalogs/catalog_item.py

```diff
@@ -1,107 +1,192 @@
+
+
+class DetailsCatalogItemView(ServicesCatalogView):
+    title = Text("#explorer_title_text")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == 'Service Catalog Item "{}"'.format(self.context['object'].name)
+        )
+
+
+class AddCatalogItemView(BasicInfoForm):
+    template_table = Table('//div[@id="prov_vm_div"]/table')
+    apply_button = Button('Apply')
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Service Catalog Item"
+        )
+
+
+class EditCatalogItemView(BasicInfoForm):
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Editing Service Catalog Item {}".format(self.name)
+        )
+
+
+class AddButtongroupView(ButtonGroupForm):
+
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Button Group"
+        )
+
+
+class AddButtonView(ButtonForm):
+
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Button"
+        )
+
+
+class EditTagsView(EditTagsForm):
+
+    save_button = Button('Save')
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == 'Editing My Company Tags for "Service Catalog Items"'
+        )
+
+
+class AddCatalogBundleView(BasicInfoForm):
+    apply_button = Button('Apply')
+    add_button = Button("Add")
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == "Adding a new Catalog Bundle"
+        )
+
+
+class EditCatalogBundleView(BasicInfoForm):
+
+    save_button = Button('Save')
+    reset_button = Button('Reset')
+
+    @property
+    def is_displayed(self):
+        return (
+            self.in_explorer and self.catalog_items.is_opened and
+            self.title.text == 'Editing Catalog Bundle "{}"'.format(self.obj.name)
+        )
 
 
 class CatalogItem(Updateable, Pretty, Navigatable):
-    pretty_attrs = ['name', 'item_type', 'catalog', 'catalog_name', 'provider', 'domain']
 
-    def __init__(self, item_type=None, vm_name=None, name=None, description=None,
-                 display_in=False, catalog=None, dialog=None,
+    def __init__(self, name=None, description=None, items=None, item_type=None,
```

Comment:
**Required**
What is items param here for? I don't see it used anywhere else and its not set as an instance var